parser-typechecker/src/Unison/Codebase/SqliteCodebase.hs

```diff
@@ -1069,27 +1073,80 @@ pushGitRootBranch srcConn branch repo = runExceptT @GitError do
       chId <- fromMaybe err <$> Q.loadCausalHashIdByCausalHash h2
       Q.setNamespaceRoot chId
 
+    -- This function makes sure that the result of git status is valid.
+    -- Valid lines are any of:
+    --
+    --   ?? .unison/v2/unison.sqlite3 (initial commit to an empty repo)
```

Comment:
I think I had seen there is sometimes a space before the `M`, will it still work if there is?

Reply:
Yes, you're right, and no, it won't work. :| I have also observed this; don't totally get the logic it's using to pick spacing.

I've pushed a fix to trim whitespace at the start of the line.

Reply:
I choose to believe `t` is short for “thank goodness we’re going to get the release out the door”